tests/queries/0_stateless/01025_array_compact_generic.reference

```diff
@@ -1,7 +1,8 @@
 []
+[1,nan,2]
 [1,nan,nan,2]
```

Comment:
``` sql
SELECT arrayCompact([1, 1, nan, nan, -nan, 2, 2, 2]);
```
Probably `-nan` should be printed here? Or there should be only one `nan`?

Reply:
-nan is equal to nan (parsed to the same bit representation)
but you can get different nans if you want by using reinterpretAsFloat64.